acquia_cms.profile

```diff
@@ -40,6 +40,8 @@ function acquia_cms_install_tasks_alter(array &$tasks) {
  */
 function acquia_cms_install_tasks(): array {
   $tasks = [];
+  $start_time = date('h:i:s a', time());
```

Comment:
@panshulK I think we need to include date month year as well. Because when you'll convert this time in unix timestamp `(strtotime())` this will assume date as today's date. So, this might result in incorrect time or negative time. 
Ex: Let's say we start installation today (i.e 18/06/2021) and install time starts at 11:45 pm and ends at 00:15 am i.e it took overall 30 mins. But when this time convert in unix timestamp, so actually it would convert following:
Start time: **18/06/2021 11:45 pm**. End time: **18/06/2021 00:15 am**, but actually it's incorrect as end time is: **19/06/2021 00:15 am**.

Reply:
Done @vishalkhode1